ProjectEuler/Problems/Problems_051_To_075.cs

```diff
@@ -123,64 +123,32 @@ public class Problems_051_To_075
 
         public static void Problem67()
         {
-            Console.WriteLine("\n\n\n67. ");
+            Console.WriteLine("\n\n\n67. Find the maximum total from top to bottom in triangle.txt,\na 15K text file containing a triangle with one-hundred rows.");
 
             DateTime startTime = DateTime.Now;
             var triangle = GetTriangle();
 
-            FindGreatestSumInTriangleMultithread(0, 0, triangle, out ulong greatestSum);
+            ulong greatestSum = FindGreatestSumInTriangle(triangle);
 
             Console.WriteLine("\n\tAnswer: " + greatestSum);
 
             Console.WriteLine("\tTook: " + (DateTime.Now - startTime));
         }
 
-        private static void FindGreatestSumInTriangleMultithread(int row, int col, List<List<ulong>> triangle, out ulong greatestSum)
+        private static ulong FindGreatestSumInTriangle(List<List<ulong>> triangle)
         {
-            // If we aren't at the bottom
-            if (row < triangle.Count() - 1)
+            // Start at the bottom of the triangle over-writing the parents with the addition of the parents and the parent's biggest child
+            for (int parentRow = triangle.Count - 2; parentRow >= 0; parentRow--)
```

Comment:
You are going to have to explain this to me!

Reply:
Ok, remind me later and I'll give you a fun white board explaination